util/regexp-assemble/data/934120.data

```diff
@@ -14,8 +14,13 @@
 ##! - `##!$`: the suffix comment
 ##! Please refer to util/regexp-assemble/README.md for a full explanation.
 
-##! Plenty of schemes/protocols to catch here (from https://en.wikipedia.org/wiki/List_of_URI_schemes):
-##!^(?i)(?:acap|bitcoin|blob|cap|cvs|svn|svn\+ssh|turn|udp|vnc|xmpp|webcal|xri|dav|dns|feed|finger|h323|icap|ipp|ipps|jabber|mms|mumble|maven|nfs|rmi|rsync|rtmp|rtm|rtmfp|http|https|ftp|ftps|git|gopher|irc|fd|ldap|ldapi|ldaps|expect|netdoc|jar|ogg|tftp|dict|ssh2|ssh|imap|pop2|pop3|mailto|nntp|news|snews|snmp|smtp|sftp|telnet|s3|phar|sip|sips|smb)://
```

Comment:
Missing case flag: `##!+ i`

Reply:
Was in the other file, but moving it here for clarity.